app/views/drivers/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<h2>Driver's name: <%= @driver.name %></h2>
+<h5>VIN: <%= @driver.vin%></h5>
+<% if @driver.available %>
+  <h5>Status: Available</h5>
+<%else%>
+  <h5>Status: Unavailable</h5>
+<%end%>
+<h5>Total Earning: <%= @driver.total_earning %></h5>
+<h5>Average Rating: <%= @driver.average_rating %></h5>
+
+<button type="button"><%= link_to "Edit This Driver", edit_driver_path(@driver.id)%></button>
+<button type="button"> <%= link_to "Delete This Driver", driver_path(@driver.id), method: :delete, data: { confirm: "Are you sure you want to delete #{@driver.name}? You can't undo this action." } %></button>
+<button type="button"><%= link_to "Change Driver's Availibity", change_availability_path(@driver.id), method: :patch %></button>
+<% #TODO: Ask Alicia if she knows how to use ternary to change %>
+
+<% #TODO! list of trips goes here %>
+<h3>Driver's Trips</h3>
+<% if @driver.trips.length > 0 %>
+  <table class="table">
+    <tr>
+      <th>Trip's ID</th>
+      <th>Trips's Date</th>
+      <th>Passenger's Name</th>
+      <th>Driver's Name</th>
+      <th>Rating</th>
+      <th>Cost</th>
+    </tr>
+  <% @driver.trips.each do |trip| %>
+  <tr>
+    <td><%= link_to trip.id, trip_path(trip.id) %></td>
+    <td><%= trip.date %></td>
+    <td><%= link_to trip.passenger.name, passenger_path(trip.passenger.id) %></td>
+    <td><%= link_to trip.driver.name, driver_path(trip.driver.id) %></td>
+    <td><%= trip.rating %></td>
+    <td>$<%= trip.cost %></td>
+  </tr>
+  <% end %>
+  </table>
+<% else %>
```

Comment:
Just noting.  This would make an excellent partial view that you can use in both the driver and passenger show page.